modules/ROOT/nav.adoc

```diff
@@ -8,14 +8,15 @@
 * xref:dashboards.adoc[Dashboards]
  ** xref:app-dashboards.adoc[Builtin Application Dashboards]
  ** xref:api-analytics-dashboard.adoc[Builtin API Analytics Dashboard]
-* xref:dashboards-using.adoc[Using Dashboards]
+ ** xref:dashboards-using.adoc[Using Dashboards]
  ** xref:dashboard-custom-config.adoc[Configuring Custom Dashboards and Charts]
  ** xref:dashboard-custom-config-graph.adoc[Configuring Graphs]
  ** xref:dashboard-custom-config-singlestat.adoc[Configuring Singlestats]
  ** xref:dashboard-custom-config-table.adoc[Configuring Tables]
  ** xref:dashboard-custom-config-text.adoc[Configuring Text]
  ** xref:dashboard-config-ref.adoc[Dashboard Configuration Reference]
- ** xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector]
+* xref:anypoint-custom-metrics-connector.adoc[Custom Metrics]
+ ** xref:custom-metrics-dashboard.adoc[Use the Custom Metrics Dashboard]
```

Comment:
I wouldn't make this a child of Custom Metrics until there was a 2nd topic to go under it. YMMV.